test/ctmrg/gradparts.jl

```diff
@@ -0,0 +1,147 @@
+using Test
+using Random
+using PEPSKit
+using TensorKit
+using PEPSKit: NORTH, SOUTH, WEST, EAST, NORTHWEST, NORTHEAST, SOUTHEAST, SOUTHWEST, rotate_north, left_move, ctmrg_iter
+using Zygote
+using ChainRulesCore
+using ChainRulesTestUtils
+using FiniteDifferences
+
+## Test utility
```

Comment:
Please refactor to separate file, so it can be included from multiple places instead of being copied everywhere...